docs/whats-new/dotnet-7-docs.md

```diff
@@ -0,0 +1,295 @@
+- [dotnet new search](../core/tools/dotnet-new-search.md) - New syntax for `dotnet new`
+- [dotnet new uninstall](../core/tools/dotnet-new-uninstall.md) - New syntax for `dotnet new`
+- [dotnet new update](../core/tools/dotnet-new-update.md) - New syntax for `dotnet new`
+- [dotnet new &lt;TEMPLATE&gt;](../core/tools/dotnet-new.md) - New syntax for `dotnet new`
+- [dotnet restore](../core/tools/dotnet-restore.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [dotnet publish](../core/tools/dotnet-publish.md) - Document Publish UseCurrentRuntime
+- [dotnet build](../core/tools/dotnet-build.md) - Document UseCurrentRuntime
+- [dotnet watch](../core/tools/dotnet-watch.md) - Note that --non-interactive is .NET 7
+- [dotnet nuget sign](../core/tools/dotnet-nuget-sign.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [dotnet nuget trust](../core/tools/dotnet-nuget-trust.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [dotnet nuget verify](../core/tools/dotnet-nuget-verify.md) - Document .NET 7 SDK changes to NuGet signed package verification
+- [.NET SDK error list](../core/tools/sdk-errors/index.md)
+  - Add new errors/warnings to NETSDK error index
+  - Document selected NETSDK error messages
+  - List of NETSDK error messages
+- [NETSDK1100: Set the EnableWindowsTargeting property to true](../core/tools/sdk-errors/netsdk1100.md)
+  - Add three new properties
+  - Document selected NETSDK error messages
+- [dotnet-coverage code coverage utility](../core/additional-tools/dotnet-coverage.md) - Adding new options and example scenarios
+- [.NET additional tools overview](../core/additional-tools/index.md) - Add dotnet-monitor .NET tool documentation
+- [Diagnostics client library](../core/diagnostics/diagnostics-client-library.md) - Add dotnet-monitor .NET tool documentation
+- [Collect diagnostics in containers](../core/diagnostics/diagnostics-in-containers.md) - Add dotnet-monitor .NET tool documentation
+- [What diagnostic tools are available in .NET Core?](../core/diagnostics/index.md) - Add dotnet-monitor .NET tool documentation
+- [Source-generator diagnostics in .NET 6+](../fundamentals/syslib-diagnostics/source-generator-overview.md) - Add SYSLIB diagnostics for .NET 7 source generators
+- [Obsolete features in .NET 5+](../fundamentals/syslib-diagnostics/obsoletions-overview.md)
+  - Add two new SYSLIB warnings
+  - XmlSecureResolver breaking change
+  - Add SYSLIB0041
+  - Add SYSLIB0040
+  - Add syslib0038 and syslib0039
+- [SYSLIB0012: Assembly.CodeBase and Assembly.EscapedCodeBase are obsolete](../fundamentals/syslib-diagnostics/syslib0012.md) - Add two new SYSLIB warnings
+- [SYSLIB0043: ECDiffieHellmanPublicKey.ToByteArray is obsolete](../fundamentals/syslib-diagnostics/syslib0043.md)
+  - XmlSecureResolver breaking change
+  - Add SYSLIB0041
+- [Overview of .NET source code analysis](../fundamentals/code-analysis/overview.md) - Merge .NET 7 release branch into main
```

Comment:
Should this be included?  And should it have a different note about the change?

Reply:
No, remove it.